docs/community/release-notes.md

```diff
@@ -34,6 +34,118 @@ You can determine your currently installed version using `pip show`:
 
 ---
 
+## 3.15.x series
+
+### 3.15.0
+
+Date: 6th June 2023 
+
+* Remove dependency on pytz [[#8984](https://github.com/encode/django-rest-framework/pull/8984)]
```

Comment:
I think we need to add the following that have been merged since the last edit:

```suggestion
* Swap imports order of django-filter and CoreAPI in compat module [[#4601](https://github.com/encode/django-rest-framework/pull/4601)]
* Fix `SearchFilter` rendering search field with invalid value [[#9023](https://github.com/encode/django-rest-framework/pull/9023)]
* Remove unnecessary line which was causing isort error [[#9014](https://github.com/encode/django-rest-framework/pull/9014)]
* Update GitHub actions used in `pre-commit.yml` workflow [[#9012](https://github.com/encode/django-rest-framework/pull/9012)]
* Fix OpenAPI Schema yaml rendering for `timedelta` [[#9007](https://github.com/encode/django-rest-framework/pull/9007)]
* Fix `NamespaceVersioning` ignoring `DEFAULT_VERSION` on non-None namespaces [[#7278](https://github.com/encode/django-rest-framework/pull/7278)]
* Added Deprecation Warnings for CoreAPI [[#7519](https://github.com/encode/django-rest-framework/pull/7519)]
* Removed usage of `field.choices` that triggered full table load [[#8950](https://github.com/encode/django-rest-framework/pull/8950)]
* Permit mixed casing of string values for `BooleanField` validation [[#8970](https://github.com/encode/django-rest-framework/pull/8970)]
* Fixes `BrowsableAPIRenderer` for usage with `ListSerializer`. [[#7530](https://github.com/encode/django-rest-framework/pull/7530)]
* Change semantic of `OR` of two permission classes [[#7522](https://github.com/encode/django-rest-framework/pull/7522)]
* Remove dependency on `pytz` [[#8984](https://github.com/encode/django-rest-framework/pull/8984)]
```

Excluded:

* Revert "test codecov gha" [#8947](https://github.com/encode/django-rest-framework/pull/8947)
* test codecov gha [#8946](https://github.com/encode/django-rest-framework/pull/8946)

(sourced from `gh --repo encode/django-rest-framework pr list --state merged  --search 'sort:updated-desc' --limit 100 --json 'title,number,url'`, excluding the PRs which were already mentioned in the release notes)

Reply:
Thanks @browniebroke... 

One other point here when reviewing the changelog - we should be listing PRs which are *behavioural* changes.
I think there's at least a few items here which are internal tooling cleanups that we wouldn't need to list to our users.

Reply:
Thanks. I wasn't sure about how you handle these so I've put all of them, hoping that maintainers can make a judgement call. I think a more user facing list might be:

```suggestion
* Swap imports order of django-filter and CoreAPI in compat module [[#4601](https://github.com/encode/django-rest-framework/pull/4601)]
* Fix `SearchFilter` rendering search field with invalid value [[#9023](https://github.com/encode/django-rest-framework/pull/9023)]
* Fix OpenAPI Schema yaml rendering for `timedelta` [[#9007](https://github.com/encode/django-rest-framework/pull/9007)]
* Fix `NamespaceVersioning` ignoring `DEFAULT_VERSION` on non-None namespaces [[#7278](https://github.com/encode/django-rest-framework/pull/7278)]
* Added Deprecation Warnings for CoreAPI [[#7519](https://github.com/encode/django-rest-framework/pull/7519)]
* Removed usage of `field.choices` that triggered full table load [[#8950](https://github.com/encode/django-rest-framework/pull/8950)]
* Permit mixed casing of string values for `BooleanField` validation [[#8970](https://github.com/encode/django-rest-framework/pull/8970)]
* Fixes `BrowsableAPIRenderer` for usage with `ListSerializer`. [[#7530](https://github.com/encode/django-rest-framework/pull/7530)]
* Change semantic of `OR` of two permission classes [[#7522](https://github.com/encode/django-rest-framework/pull/7522)]
* Remove dependency on `pytz` [[#8984](https://github.com/encode/django-rest-framework/pull/8984)]
```

Not entirely sure about the first one, it seems to be fixing a user facing issue, but the current wording isn't clear which one, and I couldn't figure that out.

Reply:
I will review it and let you know as per Toms suggestion.

Reply:
"Fix compatibility issues between django-filter and CoreAPI by swapping imports order in compat module" -- seems better

Reply:
> Thanks @browniebroke...
> 
> One other point here when reviewing the changelog - we should be listing PRs which are _behavioural_ changes. I think there's at least a few items here which are internal tooling cleanups that we wouldn't need to list to our users.

The criteria I was using is to include behavioral / relevant changes in the Announcement while all the changes in the Release Notes (a trimmed "urlified to github" version of ``git log --pretty=oneline HEAD...3.14.0`` - trimmed the reverts) so it can be further worked, it's great to have a clarification on the criteria :)

I commited @browniebroke's changes and rebased upon latest master so @auvipy can move forward :)